### ISSUE TITLE
Incorrect structure generated from "CIF" file for some systems

### ISSUE THREAD

**Issue by guoqianglan** [materialsproject/pymatgen#483]
<When reporting bugs/issues, please supply the following information. If
this is a feature request, please simply state the requested feature.>

## System

* Pymatgen version: 4.3.2
* Python version:  2.7
* OS version: Linux

## Summary

1. Some structures (e. g. Fe2O3, mp-19306) generated from "CIF" files are incorrect

## Example code

```python
from pymatgen.io.cif import CifWriter
material_id = "mp-19306"
structure = mpr.get_structure_by_material_id(material_id, final=True)
print structure.composition.reduced_formula
cif = CifWriter(structure, symprec=0.1)
cif.write_file(material_id+".cif")
structure2 = Structure.from_file(material_id+".cif")
print structure2.composition.reduced_formula
```

## output

```
Fe3O4
Fe5O4
```

## Description of problem

Firstly, I generated a "CIF" file from the computed structure from the materials project database. Then I regenerate a pymatgen structure object from this "CIF" file, but the structure was not correct anymore. I can make sure that the crystal information in "CIF" file is correct. There should be some bugs in "Structure.from_file" function when dealing with "CIF" file?

**Comment by setten** [comment]
I think it’s your symprec, your probably forcing a higher or lower symmetry in writing the gif.

best,
Michiel

**Comment by shyuep** [comment]
@wmdrichards Can you look into this pls?

**Comment by wmdrichards** [comment]
I agree with Michiel - the SpacegroupAnalyzer returns a nonsense structure
with symprec=0.1 (look at the written out cif).
Maybe we should warn users trying too large values for symprec? Though I
haven't figured out its exact implementation, so I don't know what would be
considered large enough to have this problem. Or we could do a composition
check on SpacegroupAnalyzer.get_refined_structure()

**Comment by guoqianglan** [comment]
@wmdrichards 
Hi, it is nothing to do with the symprec. I try to input this "CIF" file to "Vesta" to generate a new POSCAR. This POSCAR is correct with correct atomic ratio, Fe:O=3:4. But the "Structure.from_file" function didn't output a structure correctly.
By the way, the purpose I used a large symprec (0.1) is to get the symmetry of Fe3O4 back. The spacegrop of the computed structure of Fe3O4 in the material project database is "R-3m", but it should be with "Fd-3m" spacegroup. Only by using large symprec can I get the "Fd-3m"  symmetry back.
Regards 
Guoqiang

**Comment by wmdrichards** [comment]
It was a symprec issue in the cif generation. Closed with bc4096e
